packages/v4/src/content/design-guidelines/usage-and-behavior/dashboard/dashboard.md

```diff
@@ -0,0 +1,331 @@
+---
+id: Dashboards
+section: guidelines
+---
+
+A dashboard provides overviews of key metrics or performance indicators relevant to an application, process, or business. The overall experience of dashboards can vary greatly depending on their use cases,. so it can be difficult to define a one-size-fits-all approach. These guidelines take a systematic approach to dashboard design that can be applied in a variety of situations.
+
+## Elements
+While dashboards can vary greatly depending on what requirements and use cases they’re meant to fulfill, they can typically be distilled into two main parts:
+
+- **Cards:** Information is communicated via cards. Each card should be designed to convey a single metric or a group of closely related metrics.
+
+- **Layout:** The arrangement of cards, typically in a grid structure.
+
+We’ll focus on these two aspects as we move through the dashboard design process. Here is an example of a dashboard containing four cards arranged in a grid.
+
+<img src="./img/dashboard-view-desktop.png" alt="example of a dashboard" width="1500"/>
+
+## Standard cards
+PatternFly supports several common card layouts to help convey information in a consistent way. This is not an exhaustive list, but before designing your cards, you should see if the information you are trying to communicate fits into any of these common patterns.
+
+### Aggregate status card
+Aggregate status cards show a total number of objects and an aggregated status for them. For example, it can be used to provide users with a quick count of the number of nodes making up a large distributed network and identify the numbers that are down, that need maintenance, etc. There would also typically be a link to examine more details.
+
+<img src="./img/aggregate-status-cards.png" alt="aggregate status card examples" width="542"/>
```

Comment:
Would try to reduce the spacing on this image, looks  each card example look unnecessarily far apart!
![image](https://user-images.githubusercontent.com/51165119/135642658-f4ac39ec-3f9a-4216-b2a2-f0a1e0b0743b.png)